RevenueCat/Plugins/iOS/PurchasesUnityHelper.m

```diff
@@ -442,6 +445,114 @@ - (void)showInAppMessages:(NSArray<NSNumber*>*)messageTypes {
     #endif
 }
 
+- (void)getEligibleWinBackOffersForProduct:(NSString *)productIdentifier {
+    // Checking for win-back offer APIs being available in the current OS is handled at the PHC level.
+    [RCCommonFunctionality eligibleWinBackOffersForProductIdentifier:productIdentifier
+                                                        completionBlock:^(NSArray<NSDictionary *> * _Nullable eligibleWinBackOffers, RCErrorContainer * _Nullable errorContainer) {
+
+        RCErrorContainer *finalErrorContainer = errorContainer;
+        if (eligibleWinBackOffers == nil && errorContainer == nil) {
+            NSError *nsError = [[NSError alloc] initWithDomain:RCPurchasesErrorCodeDomain
+                                                            code:RCUnknownError
+                                                        userInfo:@{NSLocalizedDescriptionKey: @"Both error and response are nil"}];
```

Comment:
I believe there are some indentation issues in this file

Reply:
Good catch, fixed in d3ff240f8dd0b44f1260bc2a12b4bccd7f437083